Makefile

```diff
@@ -128,16 +129,16 @@ devtools-stamp: tools
 devtools-clean: tools-clean
 	rm -f devtools-stamp
 
-vendor-deps: tools
-	@echo "--> Generating vendor directory via dep ensure"
-	@rm -rf .vendor-new
-	@dep ensure -v -vendor-only
-	tar -c vendor/ | sha1sum | cut -d' ' -f1 > $@
+go-mod-cache: go.sum
+	@echo "--> Download go modules to local cache"
+	@go mod download
+
+go.sum: go.mod
+	@echo "--> Generating vendor directory via go mod vendor"
+	@go mod vendor
```

Comment:
We probably do not need to call `go mod vendor`. I think this is only useful if people use version <go1.11:
https://github.com/golang/go/wiki/Modules#how-do-i-use-vendoring-with-modules-is-vendoring-going-away

Reply:
Removed